com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Added
 
 - Added "Check for NetworkObject Component" property to the Multiplayer->Netcode for GameObjects project settings. When disabled, this will bypass the in-editor `NetworkObject` check on `NetworkBehaviour` components. (#3034)
+- Added message size validation to named and unnamed message sending functions for better error messages. (#2363)
```

Comment:
Should be #3043 in the changelog

Reply:
oh thought it was the issue number, fixing